bigtable/integration_test.go

```diff
@@ -1998,9 +2004,29 @@ func TestIntegration_AdminBackup(t *testing.T) {
 		t.Skip("emulator doesn't support backups")
 	}
 
+	iAdminClient, err := testEnv.NewInstanceAdminClient()
+	if err != nil {
+		t.Fatalf("NewInstanceAdminClient: %v", err)
+	}
+	defer iAdminClient.Close()
+
+	// Create a development instance
+	conf := &InstanceConf{
+		InstanceId:   instanceToCreate,
+		ClusterId:    clusterID,
+		DisplayName:  "test instance",
+		Zone:         instanceToCreateZone,
+		InstanceType: DEVELOPMENT,
+		Labels:       map[string]string{"test-label-key": "test-label-value"},
+	}
 	timeout := 5 * time.Minute
 	ctx, _ := context.WithTimeout(context.Background(), timeout)
 
+	if err := iAdminClient.CreateInstance(ctx, conf); err != nil {
+		t.Fatalf("CreateInstance: %v", err)
+	}
+	defer iAdminClient.DeleteInstance(ctx, instanceToCreate)
+
```

Comment:
Only for testing

Reply:
Not sure I'm getting. You're creating an instance, but I don't see where it is used next.

Reply:
It's used for command-line setting, without it there will be an error: Instance not set

Reply:
![image](https://user-images.githubusercontent.com/38891896/99381654-6f4a7980-28dc-11eb-9385-08d9cf5b1eb9.png)

Reply:
I won't add this in production code